qutip/tests/solver/test_brmesolve.py

```diff
@@ -0,0 +1,294 @@
+import pytest
+import numpy as np
+import qutip
+from qutip.solver.brmesolve import brmesolve
+
+
+def pauli_spin_operators():
+    return [qutip.sigmax(), qutip.sigmay(), qutip.sigmaz()]
+
+
+_simple_qubit_gamma = 0.25
+coeff = qutip.coefficient(lambda t, w: _simple_qubit_gamma * (w >= 0),
+                          args={'w':0})
+_m_c_op = np.sqrt(_simple_qubit_gamma) * qutip.sigmam()
+_z_c_op = np.sqrt(_simple_qubit_gamma) * qutip.sigmaz()
+_x_a_op = [qutip.sigmax(), coeff]
+
+
+@pytest.mark.parametrize("me_c_ops, brme_c_ops, brme_a_ops", [
+    pytest.param([_m_c_op], [], [_x_a_op], id="me collapse-br coupling"),
+    pytest.param([_m_c_op], [_m_c_op], [], id="me collapse-br collapse"),
+    pytest.param([_m_c_op, _z_c_op], [_z_c_op], [_x_a_op],
+                 id="me collapse-br collapse-br coupling"),
+])
+def test_simple_qubit_system(me_c_ops, brme_c_ops, brme_a_ops):
+    """
+    Test that the BR solver handles collapse and coupling operators correctly
+    relative to the standard ME solver.
+    """
+    delta = 0.0
+    epsilon = 0.5 * 2 * np.pi
+    e_ops = pauli_spin_operators()
+    H = delta * 0.5 * qutip.sigmax() + epsilon * 0.5 * qutip.sigmaz()
+    psi0 = (2 * qutip.basis(2, 0) + qutip.basis(2, 1)).unit()
+    times = np.linspace(0, 10, 100)
+    me = qutip.mesolve(H, psi0, times, c_ops=me_c_ops, e_ops=e_ops).expect
+    brme = brmesolve(H, psi0, times, brme_a_ops,
+                     e_ops=e_ops, c_ops=brme_c_ops).expect
+    for me_expectation, brme_expectation in zip(me, brme):
```

Comment:
We should do `zip(me.expect, brme.expect)` here for consistency with the other tests.